easybuild/easyconfigs/t/TomoPy/TomoPy-1.11.0-foss-2020b.eb

```diff
@@ -0,0 +1,53 @@
+# Authors: Joachim Hein, Zdenek Matej, Lund University
+
+easyblock = 'PythonPackage'
+
+name = 'TomoPy'
+version = '1.11.0'
+
+github_account = 'tomopy'
+homepage = "https://tomopy.readthedocs.io/en/latest/"
+description = """
+Tomographic reconstruction creates three-dimensional views of an object by 
+combining two-dimensional images taken from multiple directions, for example, 
+this is how a CAT (computer-aided tomography) scanner generates 3D 
+views of the heart or brain.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('scikit-image', '0.18.1'),
+    ('OpenCV', '4.5.1', '-contrib'),
+    ('imkl', '2020.4.304'),
+    ('numexpr', '2.8.1'),
+    ('DXchange', '0.1.8'),
+    ('PyWavelets', '1.1.1'),
```

Comment:
I can't actually find that these are actually dependencies of tomopy looking through setup.py and CMakeLists

Reply:
Hi @Micket, thanks for the comment. When we started thinking about this software we were aware of the fact that Conda is the most common distribution for this software. There is some sw engineering on [TomoPy](https://github.com/tomopy/tomopy) so I guess you are right and TomoPy can be build and installed without some dependencies. On the other side TomoPy is that type of software that it installs without some dependencies but some functionality is missing. We have tried to create a similar comfortable installation as with Conda so when defining dependencies we were looking also in its conda env recipes as e.g. [linux-38.yml](https://github.com/tomopy/tomopy/blob/master/envs/linux-38.yml). In particular `DXchange` is practically an I/O module for TomoPy. And it is very comfortable for the user to have it as the user can run immediately e.g. the [tooth example](https://github.com/tomopy/tomopy/blob/master/doc/source/ipynb/tomopy.ipynb).

So I suggest to keep these dependencies. An alternative is that we provide something like TomoPy-suite but that makes things just complicated.